features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx

```diff
@@ -80,6 +81,7 @@ const styles = theme => ({
         borderBottom: 'solid 1px ' + theme.palette.grey.A200,
         fontSize: '11px',
         paddingLeft: theme.spacing.unit,
+        height: 35,
```

Comment:
Shall we use `theme.spacing.unit,` to define height here ?

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6728